### PR TITLE
10 Initialize Static Analysis Tools

### LINKED ISSUE #10: Initialize the Static Analysis Tools
Initialize the Static Analysis Tools that resulted from the research from Sprint #1 by trying them out and add additional files to Git

### PR DESCRIPTION
**Pull request by mdingjan**
Will close #10 

The following Static Analysis Tools, that have been analysed in the SAT research document, have been installed / added to the readme file:
- TSLint
- MaDGe (dependency tool)

The rule set of TSLint is defined within the 'tslint.json' file.

When reviewing this branch, it is useful to actually perform the steps.
If you have any remarks, please let me know. :)

(P.S. TSLint is also integrated to our Travis CI.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,39 @@ All documents can in general be found in the doc folder.
...
+
+### Tooling (SAT)
+**Installing and activating TSLint:**
+- Within WebStorm, open the Settings / Preferences Dialog by
+  - (for Windows and Linux) clicking on the 'File' tab and then 'Settings'.
+  - (for OS X) clicking on the 'WebStorm' tab and then 'Preferences'.
+  - the shortcut: ctrl + alt + s.
+- Go to 'Languages & Frameworks' and click on 'Node.js and NPM'.
+- For the 'Packages' area that shows up, click '+'.
+- In the 'Available Packages' dialog box that opens, select the 'tslint' package and click 'Install package'.
```
Comment (mpsijm) [inline README.md]: This entire paragraph can be replaced by `npm install -g tslint` :)

**Comment by mdingjan** [comment]
@mpsijm Thanks for the comment! I took it into account. :)

**Comment by mpsijm** [comment]
Okay, looks great now :) Ready for merge!

**Comment by thervh70** [comment]
Should not be merged into master, please change the base or open a new PR.

**Comment by mdingjan** [comment]
I've closed this PR because the base was master instead of dev.

**Pull request closed without merging**